### PR TITLE
Static content completed for all six bemisia tabaci e103. 

### PR DESCRIPTION
**Pull request by ens-LCampbell**
Ensembl metazoa 103 is the first initial release for the Bemisia tabaci sp. (all belong to cryptic species complex). Species image files contain the same exact image with specific GCA assembly accession appended to each.

### CODE REVIEW

File: metazoa/species/Bemisia/Bemisia_tabaci/Bemisia_tabaci_gca903994095_assembly.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Assembly**
+------------------------
+The *Bemisia tabaci* Uganda 1 genome was produced by the genomics consortium of the [African Cassava Whitefly Project](http://www.cassavawhitefly.org), funded by the Bill & Melinda Gates Foundation (Grant Number OPP1058938). 
+
+A field-collected colony (Namulonge, Uganda) was established, maintained and inbred \(F6 generation\) by [Dr Habibu Mugerwa](https://scholar.google.com/citations?user=srDeBUgAAAAJ&hl=en) at the quarantine insectary facilities of the [Natural Resources Institute, University of Greenwich, United Kingdom](https://www.nri.org/).
```
Comment (azangru) [inline metazoa/species/Bemisia/Bemisia_tabaci/Bemisia_tabaci_gca903994095_assembly.md:5]: Could you please delete the backslashes before the parentheses (`\(F6 generation\)`)?

Comment (azangru) [inline metazoa/species/Bemisia/Bemisia_tabaci/Bemisia_tabaci_gca903994095_assembly.md:5]: `\(F6 generation\)` -> `(F6 generation)`

File: metazoa/species/Bemisia/Bemisia_tabaci/Bemisia_tabaci_gca903994105_assembly.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+**Assembly**
+------------------------
+The *Bemisia tabaci* Asia II 5 genome was produced by the genomics consortium of the [African Cassava Whitefly Project](http://www.cassavawhitefly.org), funded by the Bill & Melinda Gates Foundation (Grant Number OPP1058938). 
+
+A field-collected colony (Salem, Tamil Nadu, India) was established, maintained and inbred \(F5 generation\) by [Prof Maruthi Gowda](https://scholar.google.com.au/citations?user=5czCi6oAAAAJ&hl=en) at the quarantine insectary facilities of the [Natural Resources Institute, University of Greenwich, United Kingdom](https://www.nri.org/).
```
Comment (azangru) [inline metazoa/species/Bemisia/Bemisia_tabaci/Bemisia_tabaci_gca903994105_assembly.md:5]: `\(F5 generation\)` -> `(F5 generation)`

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
Backslashes to be removed as per Andrey's comments

**Comment by ens-LCampbell** [comment]
My apologies, brackets are removed. I was working with content md files passed from collaborators and I missed those brackets.

**Pull request merged**